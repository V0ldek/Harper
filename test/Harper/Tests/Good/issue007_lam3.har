fun :: Integer -> Integer -> Integer -> Integer -> Integer;
fun x y = {
  (x :: Integer) = x * y;
  return (\(n :: Integer) (y :: Integer) => x + n + y);
};

main :: Integer;
main = fun 7 13 17 42;