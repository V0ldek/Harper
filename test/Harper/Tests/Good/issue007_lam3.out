Execution ended with value: 150